app/views/users/show.html.erb

```diff
@@ -0,0 +1,54 @@
+<h2>User info:</h2>
+<%= @user.username %>
+<br>
+
+<h2>User events:</h2>
+<% @user.events.each do |event| %>
```

Comment:
**REQUIRED**
* The user's show page displays events the user has created and events that the user is attending to separated in past and upcoming ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))